operations/patch.go

```diff
@@ -201,6 +201,9 @@ func Patch() cli.Command {
 			if hasVariants && !hasTasks {
 				grip.Warningf("warning - you specified variants without specifying tasks")
 			}
+			if hasVariants && hasTasks {
+				grip.Infof("Submitting patch with locally defined tasks and variants")
```

Comment:
Wouldn't this also log when the user manually enters the from the command line, like `evergreen patch -v ubuntu2204 -t test-util`? If so, is there a more specific way to identify that the variants/tasks are coming from the user's local Evergreen config rather than the command line?

Maybe we can use `params.isUsingLocalAlias` and in addition, set `isUsingLocalVariants`/`isUsingLocalTasks` in a similar fashion to determine that it's their local config.

Reply:
The ticket creator's main problem seemed to be that we were using default tasks without them knowing.  They were just using `evergreen patch` but somehow got the "no variants" error and was wondering how that happened. The new change just lets users know that default tasks are being used which should solve this problem.